airflow/www/templates/airflow/dag_code.html

```diff
@@ -18,18 +18,25 @@
 {% extends "airflow/dag.html" %}
 {% block title %}Airflow - DAGs{% endblock %}
 
+{% block head_css %}
+  {{ super() }}
+  <meta name="demo-mode" content="{{ demo_mode }}">
+{% endblock %}
+
 {% block content %}
     {{ super() }}
     <h4>{{ title }}</h4>
     {{ html_code|safe }}
 {% endblock %}
 
 {% block tail %}
+    <script src="{{ url_for_asset('utils.js') }}"></script>
     {{ super() }}
     <script>
+    const DEMO_MODE = getMetaValue('demo-mode');
       // We blur task_ids in demo mode
     $( document ).ready(function() {
-      if ("{{ demo_mode }}" == "True") {
+      if (DEMO_MODE == "True") {
```

Comment:
Would this one make more sense as a css class somewhere? Also does this "demo mode" just mean we blur things in the client, but the full code is still in the source? Seems kind of a useless feature if so! :D

Reply:
This feature is useful for conferences and presentations. You can show the complexity of your DAGs, but do not show what they do. I agree that the usability of this feature is very limited and we should think about deleting it.

Reply:
I will move the CSS styles to a file. I will transfer CSS styles to a file now. Another small step to clean up CSS / JS - separate CSS and JS.

Reply:
Oh yes, it's useful for that case, good point.